### PR TITLE
DOCS: Adding some new exercise

### PR DESCRIPTION
**Pull request by SusheelThapa**
Exercise are added for concatentation, conditional statements, constants and functions.

### CODE REVIEW

File: en/exercises/multiplication.md [hunk 1]
```diff
@@ -16,8 +17,10 @@ Here, we stored the product of `3 * 2` into a `resultingValue` variable.
 
 ### 📝 Tasks:
 
-* [ ] Write a program to store the product of  `23` times `41`  and print its value.
+- [ ] Write a program to store the product of `23` times `41` and print its value.
+
+- [ ] Write a program that generates a multiplication talbe for a specific number. The program should take a number as input and then display the multiplication table for that number, from 1 to 10.
```
Comment (sumn2u) [inline en/exercises/multiplication.md]: I think we have a typo in `multiplication talbe`.

**Review by sumn2u — CHANGES_REQUESTED** [review]
Thanks for adding additional exercises. Everything looks good beside the typo.

**Comment by SusheelThapa** [comment]
@sumn2u Can you review again?

**Review by sumn2u — APPROVED** [review]
LGTM

**Pull request merged**